### PR TITLE
Format Utility csproj with updated codeformatter

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Related #4708.

Format `Microsoft.PowerShell.Commands.Utility.csproj` by codeformatter with default options.

The codeformatter is used that I compiled with newest Roslyn packages (version 2.8.2).

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [x] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/powershell/7263)

### CODE REVIEW

**Review by anmenaga — APPROVED** [review]

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs [hunk 1]
```diff
@@ -444,25 +444,25 @@ internal static string DecodeStream(Stream stream, ref Encoding encoding)
                 isDefaultEncoding = true;
             }
 
-            string content = StreamToString (stream, encoding);
+            string content = StreamToString(stream, encoding);
             if (isDefaultEncoding) do
-            {
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs]: This formatting change looks odd to me.  Would it be better to wrap the `do..while` in braces?

```c#
if (...)
{
  do
  {
  }
  while ()
}
```

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs (hunk 1 above)
Comment (iSazonov) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs]: CodeFormatter keeps previous formatting for `if-do` but remove extra braces in `do-while`.
So formally you ask other change - manual vs automatic formatting.

I agree that you request is right and I'll manually do the fix.

**Review by TravisEz13 — APPROVED** [review]
> Reviewed 31 of 31 files at r1.
> *[Reviewable](https://reviewable.io/reviews/powershell/powershell/7263)* status: :shipit: complete! all files reviewed




Approval is always assuming you address other's comments

**Comment by iSazonov** [comment]
@SteveL-MSFT Your comment was addressed.

**Review by SteveL-MSFT — APPROVED** [review]

**Review by TravisEz13 — APPROVED** [review]
> Reviewed 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/powershell/powershell/7263)* status: :shipit: complete! all files reviewed

**Pull request merged**